frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     });
     return (
       <>
-        {!loading && (
-          <div className="co-m-pane__filter-bar co-m-pane__filter-bar--with-help-text">
-            <div className="co-m-pane__filter-bar-group co-m-pane__filter-bar-group--help-text">
```

Comment:
`co-m-pane__filter-bar-group--help-text` can be removed from the SCSS.

`co-m-pane__filter-bar-group--badge` can be removed from the SCSS as well (was previously orphaned).